Source/Core/Common/Assert.h

```diff
@@ -15,7 +15,7 @@
   {                                                                                                \
     if (!(_a_))                                                                                    \
     {                                                                                              \
-      if (!PanicYesNo(_fmt_ "\n\nIgnore and continue?", __VA_ARGS__))                              \
+      if (!PanicYesNo(_fmt_, __VA_ARGS__))                                                         \
```

Comment:
What do you think about keeping the "Ignore and continue" here (on all OSes) and removing it from `ASSERT` instead? That way, code that calls `ASSERT_MSG` directly would still get the "Ignore and continue" line.

Reply:
I agree, I just changed it this way because I'd rather steer clear from non-Windows changes as I can't test them. Will change.